### ISSUE TITLE
Docker volume doesn't separate mutable and immutable files

### ISSUE THREAD

**Issue by drewhershey** [sneezymud/sneezymud#465]
### Background:
The top-level lib folder that's required for the game to function currently contains two types of files:

1) Mutable files: those that are written to frequently while the game is running. Things like player files and rent files.
2) Immutable files: those that are generally only read at boot time and loaded into memory, and changed via modifying them directly and committing the changes to the repo. Things like zonefiles, news file, and help files.

The `docker-compose.yml` file in the sneezymud-docker repo declares a single Docker volume called sneezy-lib and mounts it in the `sneezy` container at `/home/sneezy/lib`. This volume contains the entire lib folder while the container is running.

### The Problem
Docker volumes mount over top of any files at the same location in the container, and for all intents and purposes make them invisible to processes. When the volume was first set up the lib folder would have had to be manually copied into the volume to make the immutable files available to the game.

This works fine for mutable files, as they're only ever written to by the game itself, which always has access to the contents of the volume. Therefore mutable files are kept up to date inside the volume automatically.

However, while the volume is mounted, the old copy of the immutable files in the volume override any newer copies of those files in the /home/sneezy/lib folder inside the container itself (which gets copied over from the repo when the Docker image is built). Therefore any changes made to immutable files via modifying them and committing them to the repo *are* pulled into the container when a new image is built but aren't actually seen by the game, as it can only see the old versions inside the Docker volume.

This means that to have any updates to immutable files reflected in game we're currently forced to SSH into the server and manually copy any changed files into the Docker volume, from which point that will be the version the game sees. Not a great system, especially when updating lots of files at once.

### Proposed Solution
First Step:
- [x] Submit a PR to the sneezymud-docker repo that updates the docker-compose.yml and docker-compose-compile.yml files to include a new volume alongside the existing one: `sneezy-mutable:/home/sneezy/lib/mutable`. Immediately pull this new image down to the server and restart the containers so the new volume is created and ready to go. This won't affect anything yet, as nothing in the code will be referencing that filepath.

At some point thereafter, whenever it's a good time to wizlock the game for a bit:
- [x] Merge #457 to the sneezymud repo. It updates all file paths for mutable files in the code to reference mutable/* (mutable/player, mutable/account, etc). Don't pull it down to the server immediately, though.
- [x] Copy over all mutable files/folders from sneezy-lib to sneezy-mutable using `docker cp`.  There will still be a `player` folder in the top level lib folder as well, because there are some immutable dummy accounts/characters in there used by the unit tests.
    - List of mutable folders/files that will be moved:
/lib/account/
/lib/corpses/
/lib/immortals/
/lib/player/
/lib/rent/
/lib/roomdata/
/lib/mobdata/repairs/
/lib/txt/stats
/lib/txt/stats.bak

    - New structure inside mutable folder:
/lib/mutable/account/
/lib/mutable/corpses/
/lib/mutable/immortals/
/lib/mutable/player/
/lib/mutable/rent/
/lib/mutable/roomdata/
/lib/mutable/repairs/
/lib/mutable/stats
/lib/mutable/stats.bak

- [x] Now run `docker pull sneezymud/sneezymud` to pull the image created from this PR, then `docker-compose up --force-recreate --no-deps sneezy` to recreate and restart just the sneezy container using the new image. Then confirm everything works.
- [x] Submit a PR to sneezymud-docker that removes the sneezy-lib volume from the `docker-compose.yml` file. Delete the volume via `Docker volume` command.
- [x] Update the readme to reflect the need for the lib/mutable folder's existence, as this folder will have to be created manually for non-Docker builds
- [x] Update the backup script (located at /usr/local/bin/sneezymudbackup.sh on prod server) to account for the new folder structure

### Expected Results
Once this is done, mutable files will function identically to before. Any updates to immutable files by modifying them in a text editor and getting the changes merged to the master branch will now automatically reflect in the game simply by building and publishing a new Docker image, pulling the image down to the server, and restarting the container just as it works for code updates.

I've already done all of this on my own computer's local Docker installation and everything was working fine afterwards based on my testing. Please let me know if I overlooked anything. If everyone's OK with the plan and the necessary PRs are merged it shouldn't take more than 30 minutes or so to implement, once started.
